scripts/check_copyright_notice.py

```diff
@@ -28,20 +29,40 @@
 It is assumed the script is run from the repository root.
 """
 
+import argparse
 import itertools
 import re
+import shutil
+import subprocess  # nosec
 import sys
+from datetime import datetime
 from pathlib import Path
+from typing import Dict, Iterator, Optional, Tuple, cast
 
 
-SUPPORTED_YEARS_FETCHAI = ["2019", "2020", "2021"]
+CURRENT_YEAR = datetime.now().year
+VALORY_FORK_DATE = datetime.strptime(
+    "Sun Nov 01 00:00:00 2021 +0000", "%a %b %d %X %Y %z"
```

Comment:
https://github.com/valory-xyz/open-aea/tree/18895543920c1eba235a271227564a9b7fba3436 it was 30th october afaict